spec/models/comment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] I suggest that any custom methods in the models should have a corresponding unit test so please ensure that you add a test for the `increment_post_comments_counter` method in your comment model :pray: